### PR TITLE
Wine 1.7.41 (Devel)

### PR DESCRIPTION
**Pull request by uyjulian**
notes:
change sha1 to sha256
updated wine gecko/mono
remove --disable-win16 (it isn't broken for a long time anymore)

### CODE REVIEW

**Comment by tdsmith** [comment]
Hmm:

> clang: error: unable to execute command: Segmentation fault: 11
clang: error: linker command failed due to signal (use -v to see invocation)
winegcc: clang failed

Kicked off the build again.

**Comment by uyjulian** [comment]
That's weird, it worked on my OS X 10.10.3 system, no errors occurred.

**Comment by Dreysman** [comment]
Hello uyjulian, I edited my file locally to match your pull request and mine also built just fine and worked.

File: Library/Formula/wine.rb [hunk 1]
```diff
@@ -116,7 +116,6 @@ def install
     ENV.libxml2
 
     args = ["--prefix=#{prefix}"]
-    args << "--disable-win16" if MacOS.version <= :leopard
```
Comment (mikemcquaid) [inline Library/Formula/wine.rb]: You've tested this on 10.5 or below?

Comment (Dreysman) [inline Library/Formula/wine.rb]: No, not on 10.5 or below.

Comment (mikemcquaid) [inline Library/Formula/wine.rb]: I'd rather you or someone did that before we removed this, thanks.

Comment (uyjulian) [inline Library/Formula/wine.rb]: Flip the sign?
`if MacOS.version <= :leopard`
means leopard, tiger, panther, etc
or leopard, snow leopard, lion, mountain lion, mavericks, etc
I would think it's the latter.

**Comment by jacknagel** [comment]
8cab1ca4a9199ed11011a25905b314703d73a901

**Pull request closed without merging**